app/services/post_check_availability_task_service.rb

```diff
@@ -20,7 +19,10 @@ def process
   private
 
   def update_source
-    @source.update!(:info => @options[:output])
+    update_opts = {:info => @options[:output], :availability_status => source_status, :last_checked_at => @task.created_at}
```

Comment:
@hsong-rh Even the info should be updated only if the status is ok otherwise we will put bad in there